bitswap/client/internal/session/session.go

```diff
@@ -304,32 +305,44 @@ func (s *Session) run(ctx context.Context) {
 	for {
 		select {
 		case oper := <-s.incoming:
+			var opCtx context.Context
+			var span trace.Span
 			switch oper.op {
 			case opReceive:
 				// Received blocks
+				opCtx, span = internal.StartSpan(ctx, "Session.ReceiveOp")
```

Comment:
Most of these spans are frequently so small they could be events. I was trying to treat things fairly equally here since bugs should show up in any number of places. However, given this is an event loop and should be pretty tight I'd expect all the times to be almost 0.

I'm considering shrinking everything but the most obvious longer tasks (iirc broadcast and periodic search)

Reply:
I switched this to events so we didn't have to ignore `opCtx` for the linter and to shrink the number of spans